test/models/report_test.rb

```diff
@@ -20,12 +18,12 @@ def setup
   end
 
   test '時間を表示する' do
-    assert_equal 'Wed, 03 Jan 2024'.to_date, @report.created_on
+    assert_equal Time.current.to_date, @report.created_on
```

Comment:
```suggestion
    assert_equal Date.current, @report.created_on
```

の方がシンプルですね。

ただ、以下のように書いた方が`created_on`メソッドの仕様が理解しやすい（`created_at`と連動していることをイメージしやすい）と思います。

```ruby
@report.created_at = '2024-01-03 09:30'.in_time_zone
assert_equal '2024-01-03'.to_date, @report.created_on
```

Reply:
> created_onメソッドの仕様が理解しやすい

確かにこれだと、Rubyを知らない人でもイメージしやすそうですね！👍